baseapp/baseapp_test.go

```diff
@@ -39,39 +39,31 @@ var (
 
 type paramStore struct {
 	db *dbm.MemDB
+	t  *testing.T
 }
 
 func (ps *paramStore) Set(_ sdk.Context, key []byte, value interface{}) {
 	bz, err := json.Marshal(value)
-	if err != nil {
-		panic(err)
-	}
-
-	ps.db.Set(key, bz)
+	require.NoError(ps.t, err)
+	err = ps.db.Set(key, bz)
+	require.NoError(ps.t, err)
 }
 
 func (ps *paramStore) Has(_ sdk.Context, key []byte) bool {
 	ok, err := ps.db.Has(key)
-	if err != nil {
-		panic(err)
-	}
-
+	require.NoError(ps.t, err)
```

Comment:
```suggestion
	require.NoError(ps.t, err)

```

Reply:
I don't see any change here.

Reply:
`wsl` is another `golangci-lint`'s pedantic linter.

Reply:
Ah,... well there is a huge difference between two. I don't understand why do you put errcheck down to the same level as wsl.

* errcheck - Errcheck is a program for checking for unchecked errors in go programs. These unchecked errors can be critical bugs in some cases
* wsl - Whitespace Linter - Forces you to use empty lines!